### ISSUE TITLE
80vh behaves strangely in Firefox

### ISSUE THREAD

**Issue by johnjago** [thesephist/merlot#2]
I'm a big fan of markdown editors. Don't know if you've seen either https://nota.md/ or https://inns.studio/mak, but they both put a unique twist on markdown writing apps.

There are some funny issues with Firefox and the `padding-bottom: 80vh`. On Firefox 78 I get

![image](https://user-images.githubusercontent.com/19847480/117555565-e956f200-b025-11eb-86dc-9471d1272a14.png)

And on Firefox 88 (latest macOS) the editor is visible, but it doesn't have the intended spacing at the bottom. Works well on Safari and Chrome.

**Comment by thesephist** [comment]
Hi there! It seems that this is a longstanding, well-known bug in Firefox. Here's a [bug report from a couple of decades ago](https://bugzilla.mozilla.org/show_bug.cgi?id=157846) and a [ticket with more recent updates and movement](https://bugzilla.mozilla.org/show_bug.cgi?id=1099204) from a few months ago.

There's also a related bug filed against Firefox for placement of padding in overflowing elements, [ticket here](https://bugzilla.mozilla.org/show_bug.cgi?id=748518), which seems relevant.

TLDR that I could gather from my reading is

1. The issue is not as severe on newer FF releases (on my macOS, FF 88) -- the padding's gone, but content is visible. You also pointed this out.
2. The Firefox team is treating this incompatibility with Chrome/Safari as a bug and fixing it in the next release or two
    
    <img width="1048" alt="Screen Shot 2021-05-09 at 01 35 22" src="https://user-images.githubusercontent.com/5403917/117561675-ddd7eb00-b066-11eb-9580-cacc78f30eda.png">

3. There seems to be a newly added fix to upcoming versions of Firefox (comment was 3 months old?) that will make Firefox match behavior with Webkit and Chrome.

Because of that, and because Merlot is primarily for my own use, I don't feel particular inclined to try to find a workaround to support Firefox's weird behavior (especially when it seems like Firefox might be shipping a fix soon -- it seems like GitHub is also affected by this bug). I feel slightly less guilty for this than I normally would, because it's still perfectly usable on modern versions of Firefox, albeit a little un-aesthetic :)

If you feel strongly, feel free to fork the repo and add a workaround to placate Firefox! I'm happy to answer questions to help you tweak it. If it works well, I might pull it in, but no guarantees.

---

On a different note, thanks for the recs on those Markdown editors. they look great. They're both local editors, though, and one of the big reasons I built Merlot is so I could edit from my iPad and revise on my laptop, and have a web app. (Another reason is the satisfaction of using your own tools.)

Certainly great inspiration though, thanks!
